src/components/Maps/SearchBar.tsx

```diff
@@ -31,11 +31,13 @@ const useStyles = makeStyles({
 });
 
 const SearchBar = ({
+  editAddress,
   onComplete,
 }: {
+  editAddress?: string;
   onComplete: ({ latitude, longitude, address }) => void;
 }) => {
-  const [address, setAddress] = useState('');
+  const [address, setAddress] = useState('' || editAddress);
```

Comment:
nit: default states should go last
```suggestion
  const [address, setAddress] = useState(editAddress || '');
```